org_fedora_oscap/rule_handling.py

```diff
@@ -655,13 +675,21 @@ def eval_rules(self, ksdata, storage, report_only=False):
                                         common.MESSAGE_TYPE_INFO, msg))
 
         # now do the same for the packages that should be excluded
-
         # add messages for the already excluded packages
         for pkg in self._removed_pkgs:
-            msg = _("package '%s' has been added to the list of excluded "
-                    "packages" % pkg)
-            messages.append(RuleMessage(self.__class__,
-                                        common.MESSAGE_TYPE_INFO, msg))
+            if self._package_is_essential(pkg, ksdata.packages):
+                msg = _(
+                    "package '{package}' has been added to the list "
+                    "of excluded packages, but it can't be removed "
+                    "from the current software selection without breaking it."
```

Comment:
```suggestion
                    "from the current software selection without breaking the installation."
```

Reply:
Also, what do you think about mentioning profile tailoring in case user really wants to use the selected profile?

Reply:
Something like this: https://access.redhat.com/solutions/3755211

I know it can be troublesome, so I would also be fine without having it

Reply:
Good catch with the message. I would keep it simple for now though. It is more advisable to change the software selection.